src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { Error } from '../../types/todo';
+
+type Props = {
+  hasError: Error;
+  setHasError: (value: Error) => void;
+};
+
+export const Notification: React.FC<Props> = ({ hasError, setHasError }) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setHasError(Error.Not);
+    }, 3000);
+  }, [hasError]);
+
+  return (
+    <div
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal', {
+          hidden: !hasError,
```

Comment:
Text is disappearing before whole notification, try fixing this in next task by adding timeout and other state